code/game/objects/objs.dm

```diff
@@ -203,7 +203,7 @@
 		step_towards(src,S)
 
 /obj/get_spans()
-	return ..() | SPAN_ROBOT
+	return list(SPAN_ROBOT)

```

Comment:
Why is the parent call being removed?

Reply:
it will absolutely never do anything in a well-designed system but i've opted to remove the whole proc